content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -453,7 +453,7 @@ If you want to use project quotas, you should:
   sudo tune2fs -O project -Q prjquota /dev/block-device
   ```
 
-* Ensure that the root filesystem (or optional runtime filesystem is
+* Ensure that the root filesystem (or optional runtime filesystem) is
```

Comment:
Thanks @pjhwa . Changes look good.
Would you have time to fix line 447, `Ensure that the the root fileystem (or optional runtime filesystem)`?
s/fileystem/filesystem/

Reply:
Yes, I will. Thanks @kbhawkey for your review.